singly_linked_list

```diff
@@ -0,0 +1,317 @@
+//ADT implementation: https://runestone.academy/ns/books/published/pythonds3/BasicDS/TheUnorderedListAbstractDataType.html
+//except for add_item as it copying append
+
+#include <stdio.h>
+#include <stdlib.h>
+
+//define MAX_SIZE; not handled but keep in mind
+
+struct Node
+{
+    int data;
+    struct Node *next;
+};
+
+static struct Node *head;
```

Comment:
Having a global object is a bad practise. Please remove this global variable, and accept any custom list

Reply:
Absolutely agree. I used it because it is static and neglected that it is global and it is bad practise